src/ui/public/doc_table/doc_table.html

```diff
@@ -1,8 +1,28 @@
 <div
   class="doc-table-container"
   ng-if="hits.length"
-  ng-class="{ loading: searchSource.activeFetchCount > 0 }">
-  <paginate ng-if="!infiniteScroll" list="hits" per-page="50" top-controls="true">
+  ng-class="{ loading: searchSource.activeFetchCount > 0 }"
+>
+  <div ng-if="!infiniteScroll">
+    <div class="kuiBar">
```

Comment:
I updated UI Framework with a new treatment for making the text gray. Can you rebase and update the markup to this:

```html
    <div class="kuiBar docTableBar">
      <div class="kuiBarSection">
        <div
          ng-if="shouldShowLimitedResultsWarning()"
          class="kuiToolBarText kuiSubduedText">
          {{ limitedResultsWarning }}
        </div>
```